### ISSUE TITLE
Problem with using instance unique configuration (ini) files

### ISSUE THREAD

**Issue by paytah232** [arkmanager/ark-server-tools#1203]
Hi there,

I seem to be having trouble ensuring that certain instances have their own custom ini. I believe I read the docs and understood that by naming the ini a particular way, and storing it in the same path at the instance .cfg file, that it would just work. From here:
![image](https://user-images.githubusercontent.com/44075588/138982801-dc0bc10f-0610-4c88-8060-880ce9fada1e.png)
Have I understood correctly the naming convention here? That the ini file goes in the same root folder where the cfg is, and is name instancename?

It doesn't appear to work currently, as the config that is being used is a general config set by me, and not the custom ini.

Can someone help me make sure I've set it up correctly please?

Before I launch the instance, I make sure that the files on the system are at least my general config:
![image](https://user-images.githubusercontent.com/44075588/138982240-23092156-1f1a-4ea5-b449-fb68a374bc89.png)

I then start the instance (using this program) and expect that the ini will be copied, and it reports that it worked:
![image](https://user-images.githubusercontent.com/44075588/138982320-171655e1-0e14-4921-be84-d56f233d55dc.png)

However this ini files are still incorrect. I know this because my general ini has a server message, whereas the custom ini does not.

This is where my instance file and custom inis are stored (the inis are exactly the same here, but different names as a test - neither work anyway).
![image](https://user-images.githubusercontent.com/44075588/138982421-6f4c5c19-951a-404d-aa21-e3f69c5b90ea.png)

And this is the line in my arkmanager that calls this instance:
![image](https://user-images.githubusercontent.com/44075588/138982509-93d2457d-2ce2-4109-9c2a-b5ded4db0077.png)

For clarity, I will also upload my arkmanager.cfg and thenewinstance.cfg
[cfg.zip](https://github.com/arkmanager/ark-server-tools/files/7422337/cfg.zip)

Here are the custom ini files as well:
[thenewisland-ini.zip](https://github.com/arkmanager/ark-server-tools/files/7422342/thenewisland-ini.zip)

**Comment by Paytah232** [comment]
@klightspeed Any ideas?

I also tried multiple servers (using different server roots), only the first started server can be connected to. Both of them and up listening, but only 1 can ever be logged in to. They both work separately of each other.

**Comment by klightspeed** [comment]
Are you giving each instance its own Game / Query / RCON ports?

**Comment by Paytah232** [comment]
Yep. Unique ports. All forwarded. All open. Plus 1 port up for each just in case, and just never allows a connection.

Anything on why the custom game inis wouldn't be working as well?
